src/api/metadata.ts

```diff
@@ -24,10 +24,15 @@ export type MetadataRow = {
 
 export type MetadataApiRes = DjangoRestApiResponse & { results: MetadataRow[] };
 
-export function usePortalMetadataAPI(apiConfig: Record<string, any>) {
+type usePortalMetadataAPIProps = {
+  additionalPath?: string;
+  apiConfig: Record<string, any>;
+};
+export function usePortalMetadataAPI({ additionalPath, apiConfig }: usePortalMetadataAPIProps) {
   return useQuery(
     ['portal-metadata', apiConfig],
-    async () => await API.get('portal', `/metadata/`, apiConfig),
+    async (): Promise<MetadataApiRes> =>
+      await API.get('portal', `/metadata/${additionalPath}`, apiConfig),
```

Comment:
Looks like it breaks `MetadataHomeCard`, `MetadataTable`, pages... The `additionalPath` is optional or, it is not required? The call resolves to `undefined`.

```
http://localhost:8000/metadata/undefined?...
```

Reply:
Arghh! Fixing it!

Reply:
Setting default value for empty string when it is missing. This should work